schemas/common-1.json

```diff
@@ -5,7 +5,7 @@
   "definitions": {
     "query": {
       "type": "string",
-      "pattern": "(?i)^(?:select|explain(?!\\s+analyze(?!\\s+select)))(?:(?:\\/\\*.*\\*\\/)|(?:[\\\"\\'][^\\\"\\']*[\\\"\\'])|(?:--.*\\n)|[^;\\*])+;$"
+      "pattern": "(?i)^(?:analyze|reindex|vacuum|select|explain(?!\\s+analyze(?!\\s+select)))(?:(?:\\/\\*.*\\*\\/)|(?:[\\\"\\'][^\\\"\\']*[\\\"\\'])|(?:--.*\\n)|[^;\\*])*;$"
```

Comment:
We don't want to include **OPTIMIZE** there to support MySQL? For example, Quay folks might want to run it following database upgrades.